
Turn 5
================================================================
warning: entity 3 was directed to use 13 halite to move south, but only 0 halite was available
At command 1 of 3:
>>> 1   m 3 s
    2   m 1 s
    3   g

warning: owned entities 1, 3 collided on cell (36, 24) as the result of moves on this turn
At command 2 of 3:
    1   m 3 s
>>> 2   m 1 s
    3   g


Turn 7
================================================================
warning: entity 5 was directed to use 15 halite to move south, but only 0 halite was available
At command 1 of 2:
>>> 1   m 5 s
    2   g


Turn 9
================================================================
warning: entity 7 was directed to use 12 halite to move east, but only 0 halite was available
At command 2 of 3:
    1   m 5 s
>>> 2   m 7 e
    3   g


Turn 10
================================================================
warning: entity 5 was directed to use 15 halite to move east, but only 14 halite was available
At command 2 of 3:
    1   m 9 e
>>> 2   m 5 e
    3   m 7 e


Turn 11
================================================================
warning: entity 9 was directed to use 11 halite to move south, but only 0 halite was available
At command 1 of 3:
>>> 1   m 9 s
    2   m 7 s
    3   m 5 e


Turn 12
================================================================
warning: entity 7 was directed to use 11 halite to move east, but only 8 halite was available
At command 2 of 3:
    1   m 5 o
>>> 2   m 7 e
    3   m 9 s


Turn 13
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 166, in <module>
    ships.remove(me.get_ship(id))
ValueError: list.remove(x): x not in list

